pytorch_lightning/core/lightning.py

```diff
@@ -914,10 +914,19 @@ def configure_optimizers(self) -> Union[
 
         If you don't define this method Lightning will automatically use Adam(lr=1e-3)
 
-        Return: any of these 3 options:
-            - Single optimizer
-            - List or Tuple - List of optimizers
-            - Two lists - The first list has multiple optimizers, the second a list of LR schedulers
+        Return: any of these 5 options:
+            - Single optimizer.
+            - List or Tuple - List of optimizers.
+            - Two lists - The first list has multiple optimizers, the second a list of LR schedulers.
+            - Dictionary, with an `optimizer` key and (optionally) a `lr_scheduler` key.
+            - Tuple of dictionaries as described, with an optional `frequncy` key.
```

Comment:
Typo (frequ**e**ncy)

Reply:
I am not sure how to edit this file, please apply the TYPO fix here.

Reply:
When you comment the very left icon is suggestion or in diff files each file has edit option...

Reply:
```suggestion
            - Tuple of dictionaries as described, with an optional `frequency` key.
```